apps/docs/pages/guides/auth/server-side-rendering.mdx

```diff
@@ -130,6 +130,11 @@ if (refreshToken && accessToken) {
   await supabase.auth.setSession({
     refresh_token: refreshToken,
     access_token: accessToken,
+    {
+      auth: {
+        persistSession: false,
+      },
```

Comment:
```suggestion
      auth: { persistSession: false },
```

I need to trigger another build on Vercel, so I'm making an inconsequential change